src/System.Data.Common/src/System/Data/Filter/BinaryNode.cs

```diff
@@ -1505,6 +1505,9 @@ internal sealed class LikeNode : BinaryNode
         internal const int match_exact = 4;    // <STR>
         internal const int match_all = 5;      // *
 
+        // WhiteSpace Chars Include : 0x9, 0xA, 0xB, 0xC, 0xD, 0x20, 0xA0, 0x2000, 0x2001, 0x2002, 0x2003, 0x2004, 0x2005, 0x2006, 0x2007, 0x2008, 0x2009, 0x200A, 0x200B, 0x3000, and 0xFEFF.
+        private static char[] TrimChars = new char[] { (char)0x20, (char)0x3000 };
```

Comment:
Nits: This should be readonly and named s_trimChars.

Reply:
Done https://github.com/dotnet/corefx/pull/36838/commits/7db79108c5254cbebaea33ff766032b253b1a217